src/Microsoft.Diagnostics.EventFlow.Core/Implementations/HealthReporters/CsvHealthReporter.cs

```diff
@@ -506,6 +506,8 @@ private void ProcessLogFileFolder()
             }
 #elif NETSTANDARD1_6
             basePath = Path.GetDirectoryName(this.GetType().GetTypeInfo().Assembly.Location);
+#elif NETSTANDARD2_0
```

Comment:
Instead of branch code here, I suggest we change the using in this file to:
```
#if NET451
using System.Web;
#else
using System.Reflection;
#endif
```
and unify the .NETStandard implementation to:
```
#else
basePath = Path.GetDirectoryName(this.GetType().GetTypeInfo().Assembly.Location);
#endif
```
It is more possible we have to target another new .NETStandard than another .NET Framework.

Reply:
Thanks! I've updated the PR.